pkg/api/mlflow/service/artifact/storage/gs.go

```diff
@@ -97,6 +111,7 @@ func (s GS) Get(ctx context.Context, artifactURI, path string) (io.ReadCloser, e
 	// 2. get object from gcp storage.
 	reader, err := s.client.Bucket(bucketName).Object(filepath.Join(prefix, path)).NewReader(ctx)
 	if err != nil {
+		fmt.Println("err:", err)
```

Comment:
looks like a remnant of some debugging

Reply:
damn. yep. removed everything else but no this. done.